src/component/mxgraph/shape/event-shapes.ts

```diff
@@ -36,19 +38,20 @@ abstract class EventShape extends mxEllipse {
   }
 
   public paintVertexShape(c: mxgraph.mxXmlCanvas2D, x: number, y: number, w: number, h: number): void {
-    this.markNonFullyRenderedEvents(c);
+    // This will be removed after implementation of all supported events
```

Comment:
ok to me to keep this commented code as we are going to need it soon for the next events we are going to support/display